src/app/components/tables/simple/terra-simple-table.component.ts

```diff
@@ -187,12 +244,14 @@ export class TerraSimpleTableComponent<D> implements OnChanges
 
     private updateHeaderCheckboxState():void
     {
-        if(this._selectedRowList.length === 0) // anything selected?
+        let selectedRows:Array<TerraSimpleTableRowInterface<D>> = this.inputRowList.filter((row:TerraSimpleTableRowInterface<D>) => row.selected === true);
```

Comment:
In eine Methode auslagern. Wurde oben genau so schon verwendet. Am besten in einen getter. Und schon hätte man wieder Zugriff auf eine `selectedRowList` 😉

Reply:
schon klar an sich auch nicht verkehrt, aber leider hat das mit der selectedRowList nicht funktioniert da es dort merkwürdige Wechselwirkungen gab